app/tests/system/setup/setup.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * Loop through the custom contexts that exist when the app starts
+ * and grab their names.
+ *
+ * To change the environment we're testing against, specify it in site.json.
+ * This should align with what the app is built against, which is declared in
+ * app/environment.js hostUrl
+ */
+
+var Site = require('../site.json');
+
+var Setup = {};
+
+var selectors = {
+    previewButton: '#authorize',
+    registeredAuctionList: '.js-auction-lists'
+};
+
+// THESE URLS DO NOT CHANGE REGARDLESS OF ENVIRONMENT
+var expected = {
+    //EXAMPLE Webviews constructed for project
```

Comment:
This can be a longer, more detailed comment. You can describe how these values are regular expressions that match a `expected` URL.